xml/depl_maintenance.xml

```diff
@@ -508,6 +508,258 @@
    </itemizedlist>
   </sect2>
 
+  <sect2 xml:id="sec.depl.maintenance.postgre-mariadb.upgrade">
+   <title>Preparing &postgres;-Based &productname; 7 for Upgrade</title>
+   <para>
+    Upgrading &productname; 7 is only possible when it uses &mariadb; deployed
+    with the Database &barcl;. This means that before you can proceed with
+    upgrading &productname; 7, you must migrate &postgres; to &mariadb; first. The following description covers several possible scenarios.
+   </para>
+   <sect3 xml:id="sec.postgre-mariadb.upgrade.scenario1">
+    <title>Non-HA setup or HA Setup with a 2-Node Cluster and
+    &postgres; Database Backend</title>
+    <para>
+     Before you proceed, make sure that the database cluster has an odd number
+     of nodes. If necessary, extend the cluster as described in <xref
+     linkend="sec.postgre-mariadb.upgrade.scenario2"/>. Install the latest
+     maintenance updates on &productname; 7. In the
+     &crow; &wi;, switch to the <guimenu>Database</guimenu> &barcl;. You should
+     see the new <systemitem>mysql-server</systemitem> role in the
+     <guimenu>Deployment</guimenu> section. Do not change the
+     <literal>sql_engine</literal> at this point. Add your &dbnode; to the
```

Comment:
Assuming that `&dbnode` expands to "database node" I think this is a bit misleading. In the HA case you won't assign and single node to the role, but a whole cluster.

Reply:
Fixed.